src/components/CommitBox.tsx

```diff
@@ -145,6 +181,9 @@ export class CommitBox extends React.Component<ICommitBoxProps> {
    * Whether a commit can be performed (files are staged and summary is not empty).
    */
   private _canCommit(): boolean {
+    if (this.props.amend) {
+      return true;
```

Comment:
Should we protect against amending without changes (as we don't allow to edit the commit message)?

```suggestion
      return this.props.hasFiles;
```